lib/plugins/filter/before_post_render/backtick_code_block.js

```diff
@@ -48,6 +49,15 @@ function backtickCodeBlock(data) {
             options.caption += `<a href="${match[3]}">${match[4] ? match[4] : 'link'}</a>`;
           }
         }
+
+        if (options.exclude_languages) {
+          if (options.exclude_languages.includes(options.lang)) {
```

Comment:
if `exclude_languages` is a string, `c` will match `cpp`. need string -> array conversion, but don't write back to options.exclude_languages, use

```js
let { exclude_languages } = options;
```

Reply:
Well, I update the code to make sure the exclude_languages will be an array, but it is only for a defensive-programming requirement. It will be better to add a schema checker for `hexo` config.